src/doc/reference.md

```diff
@@ -586,17 +603,17 @@ syntax named by _designator_. Valid designators are:
 * `ty`: a [type](#types)
 * `ident`: an [identifier](#identifiers)
 * `path`: a [path](#paths)
-* `tt`: either side of the `=>` in macro rules
+* `tt`: a "token tree": either one [token](#token) or a delimited sequence
+   of token trees
```

Comment:
You might want to add a compareason with LISP or an AST to explain more precisely what a token tree is. But maybe it would become a bit heavy.